app/pb_kits/playbook/pb_source/_source.html.erb

```diff
@@ -1,17 +1,22 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname("pb_source")) do %>
+    class: object.classname) do %>
   <div class="pb__source_layout">
-    <% if object.hide_icon == false %>
-      <%= object.type_icon %>
-      <%= object.avatar %>
+    <% if !object.hide_icon %>
+      <% if object.show_icon? %>
+        <%= pb_rails("icon_circle", props: { icon: object.type_icon_name, size: "sm" }) %>
+      <% else %>
+        <%= pb_rails("avatar", props: object.avatar) %>
+      <% end %>
     <% end %>
     <div class="pb__source_content">
-      <%= object.source %>
+      <%= pb_rails("title", props: { text: object.source, tag: "h4", size: 4 }) %>
       <div class="pb__source_value">
```

Comment:
@markiearnold - All the `"pb__source..."` classes have 2 underscores. Is this on purpose? Does it communicate something about these styles?

Reply:
It's written that way in the scss file...I think it's like that on purpose since they're inherited styles from the main pb_source classname

Reply:
Ah. Cool. Thanks!